compiler/runtime/OMRRelocationRecord.cpp

```diff
@@ -305,73 +305,33 @@ OMR::RelocationRecord::applyRelocationAtAllOffsets(TR::RelocationRuntime *reloRu
       return 0;
       }
 
-   if (reloTarget->isOrderedPairRelocation(self(), reloTarget))
```

Comment:
Why is this code being removed? Changes to RelocationRecord were not mentioned in the commit comment, I don't think...

Reply:
Hello Mark @mstoodle , this change should have happened with the previous commit. I was rewriting history, addressing @ymanton 's suggestion to remove OrderedPairRelocations unless I am using them or thinking that consuming projects will use it. I did not have enough expertise to comment on that, so I removed the code as it is makes the PR shorter and simpler to review. What is your opinion, should I keep the OrderedPairRelocations in or remove it? In the second case, I will move the change to the previous commit and it won't be visible.

Reply:
@ymanton's suggestion is a good one: I think ordered pairs needlessly complicates the relocation infrastructure (it was like that when I got there, I assure you :) ). You should remove it, but I would prefer you put that change into the appropriate commit (the one that introduces RelocationRecord in the first place).

You can easily add it back in with a separate commit "at the end" then use git rebase to reorder the commits so that it's right after the commit where you want to "undo" it, and then squash those two commits together. Then you can remove the code in a separate commit "at the end" and do the same thing to reorder it after the earlier commit, then squash them together. Hopefully that is clear enough. git rebase is your friend, just don't aim it at any body parts...

Reply:
Thank you Mark, working on this PR I have a very good idea how to rewrite history, I just needed an executive decision. Thank you for making it! I will update my branch!